kitsune/dashboards/readouts.py

```diff
@@ -1216,13 +1216,13 @@ def _format_row(self, row):
 
 # L10n Dashboard tables that have their own whole-page views:
 L10N_READOUTS = OrderedDict(
-    (t.slug, t)
+    (t.slug, t)  # type: ignore
```

Comment:
Another strange thing in `mypy`. It flags `t.slug` as an attribute error thinking that the `slug` attribute doesn't exist, even though it's defined on all of classes provided.